### PR TITLE
cal, bump version

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

File: app-misc/cal/cal-4.1.recipe [hunk 1]
```diff
@@ -0,0 +1,43 @@
+SUMMARY="Display a monthly or yearly calendar, with optional appointments"
+DESCRIPTION="By default, cal  will display a calendar for the current month \
+with the current day marked. By specifing  certain arguments, cal will \
+display a calendar for a whole year or a specified month and year."
+HOMEPAGE=""
```
Comment (alaviss) [inline app-misc/cal/cal-4.1.recipe]: No homepage?

File: app-misc/cal/cal-4.1.recipe [hunk 2]
```diff
@@ -0,0 +1,43 @@
+SUMMARY="Display a monthly or yearly calendar, with optional appointments"
+DESCRIPTION="By default, cal  will display a calendar for the current month \
+with the current day marked. By specifing  certain arguments, cal will \
```
Comment (alaviss) [inline app-misc/cal/cal-4.1.recipe]:
```diff
-DESCRIPTION="By default, cal  will display a calendar for the current month \
-with the current day marked. By specifing  certain arguments, cal will \
+DESCRIPTION="By default, cal will display a calendar for the current month \
+with the current day marked. By specifing certain arguments, cal will \
```

File: app-misc/cal/cal-4.1.recipe [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+display a calendar for a whole year or a specified month and year."
+HOMEPAGE=""
+COPYRIGHT="1993-2014, Alex Matulich, by Unicorn Research Corporation."
+LICENSE="BSD (4-clause)"
+REVISION="1"
+SOURCE_URI="http://unicorn.us.com/pub/cal41.zip"
+CHECKSUM_SHA256="96318b738970237832e4594cb363058f4243f54e86a5f1e06737fb5d02e6c298"
+SOURCE_DIR="cal41"
```
Comment (alaviss) [inline app-misc/cal/cal-4.1.recipe]: We could drop `cd src` statements with this:

```diff
-SOURCE_DIR="cal41"
+SOURCE_DIR="cal41/src"
```

Also, the [Guidelines][0] forbids using `cd` to navigate to a package's main directory, but I'm not sure if it applies here.

[0]: https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#source-directory

File: app-misc/cal/cal-4.1.recipe (hunk 1 above)
Comment (Begasus) [reply app-misc/cal/cal-4.1.recipe]: Need to check my local recipe (latest), thought I'd put it in ...

File: app-misc/cal/cal-4.1.recipe (hunk 3 above)
Comment (Begasus) [reply app-misc/cal/cal-4.1.recipe]: Nice, didn't know this one yet :)

Comment (Begasus) [reply app-misc/cal/cal-4.1.recipe]: It works :+1:

File: app-misc/cal/cal-4.1.recipe [hunk 4]
```diff
@@ -0,0 +1,41 @@
+SUMMARY="Display a monthly or yearly calendar, with optional appointments"
+DESCRIPTION="By default, cal will display a calendar for the current month \
+with the current day marked. By specifing certain arguments, cal will \
+display a calendar for a whole year or a specified month and year."
+HOMEPAGE="http://unicorn.us.com/cal.html"
+COPYRIGHT="1993-2014, Alex Matulich, by Unicorn Research Corporation."
```
Comment (fbrosson) [inline app-misc/cal/cal-4.1.recipe]: I think we prefer to drop the first comma. And maybe we should drop the `by ` too (but I'm not sure about that.)
```diff
-COPYRIGHT="1993-2014, Alex Matulich, by Unicorn Research Corporation."
+COPYRIGHT="1993-2014 Alex Matulich, Unicorn Research Corporation."
```
That said, I'm ok with what we have right now :)

Comment (Begasus) [reply app-misc/cal/cal-4.1.recipe]: @your service :)

Comment (fbrosson) [reply app-misc/cal/cal-4.1.recipe]: @alaviss, I think the PR is ready for a merge.
(All your comments have been addressed, and I have also checked that it builds on all arches :)

**Pull request merged**